pkg/googlecloud/subscriber.go

```diff
@@ -86,6 +90,15 @@ type SubscriberConfig struct {
 	Unmarshaler Unmarshaler
 }
 
+// FetchTopicProjectID returns the proper topic project ID to use.
+func (sc SubscriberConfig) FetchTopicProjectID() string {
```

Comment:
Could you make this method private? I guess that we don't need to extend public API :wink:.

I'm also thinking if just `topicProjectID()` will be not better because we are not fetching anything - it's just a getter :)